pkgs/development/ocaml-modules/ocamlformat-mlx/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  cmdliner,
+  odoc,
+  ocamlformat-mlx-lib,
+  re,
+}:
+buildDunePackage rec {
+  pname = "ocamlformat-mlx";
+  version = "0.26.2.0";
```

Comment:
We likely want inherit things from ocamlformat-mlx-lib to make sure they are not out of sync.
`inherit (ocamlformat-mlx-lib) version src`

Reply:
Not sure if minimalOcamlVersion is inheritable as well